index.html

```diff
@@ -1,86 +1,97 @@
+                        <a href="#" class="postCategory">Chuyện của tôi</a>
+                        <a href="#" class="postTitle">Lorem ipsum dolor sit amet consectetur adipiscing elit. aqwrqtqwr</a>
+                        <p class="postDescription">Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>
+                        <div class="postInfoShare">
+                            <p class="postInfo">bởi <a href="#">Pikachu</a>, 07/07/2011</p>
+                            <a href="#" class="postShare">Share</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="post">
+                    <div class="postImg">
+                        <img src="image/post_1.png" alt="postImg">
+                    </div>
+                    <div class="postMain">
+                        <a href="#" class="postCategory">Chuyện của tôi</a>
+                        <a href="#" class="postTitle">Click this card for redirect to full story</a>
+                        <p class="postDescription">Lorem ipsum dolor sit amet consectetur adipiscing elit Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>
+                        <div class="postInfoShare">
+                            <p class="postInfo">bởi <a href="#">Pikachu</a>, 07/07/2011</p>
+                            <a href="#" class="postShare">Share</a>
+                        </div>
+                    </div>
+                </div>
+                <div class="post">
+                    <div class="postImg">
+                        <img src="image/post_2.png" alt="postImg">
+                    </div>
+                    <div class="postMain">
+                        <a href="#" class="postCategory">Chuyện của tôi</a>
+                        <a href="#" class="postTitle">Click this card for redirect to full story</a>
+                        <p class="postDescription">Lorem ipsum dolor sit amet consectetur adipiscing elit Lorem ipsum dolor sit amet consectetur adipiscing elit.</p>
+                        <div class="postInfoShare">
+                            <p class="postInfo">bởi <a href="#">Pikachu</a>, 07/07/2011</p>
+                            <a href="#" class="postShare">Share</a>
+                        </div>
+                    </div>
+                </div>
+                
             </div>
-        </div>
-        <div class="subscribe">
-            <h3 class="title">Subscribe Me</h3>
-            <div class="contain-form">
-                <form action="">
-                    <input type="text" name="email" placeholder="Enter your email">
-                    <input type="submit" value="Subscribe">
-                </form>
+        </section>
+        <section class="pagination">
+            <div class="container paginateContain">
+                <a href="#" class="arrowPaginate arrowPrev"><i class="fa fa-chevron-left"></i></a>
+                <ul class="paginate">
```

Comment:
Name of class/id should be **noun**. "Paginate" is a verb.

Reply:
I deleted it. And it is a part of new task.